src/pages/Home/Home.js

```diff
@@ -1,11 +1,26 @@
 import { useHistory } from 'react-router-dom'
+import NavigationButtons from '../../components/NavigationButton/NavigationButton'
+import LastTransaction from '../../components/LastTransaction/LastTransaction'
+import styled from 'styled-components'
 
-export default function Home() {
+export default function Home({ data }) {
   const history = useHistory()
   return (
-    <main>
-      <button onClick={() => history.push('/earn-coins')}>Earn coins</button>
-      <button onClick={() => history.push('/use-coins')}>Use coins</button>
-    </main>
+    <>
+      <LastTransaction data={data} />
+      <ButtonWrapper>
+        <NavigationButtons onClick={() => history.push('/earn-coins')}>
+          Earn coins
+        </NavigationButtons>
+        <NavigationButtons onClick={() => history.push('/use-coins')}>
+          Use coins
+        </NavigationButtons>
+      </ButtonWrapper>
+    </>
   )
 }
+
+const ButtonWrapper = styled.main`
```

Comment:
It's a navigation, why not use `<nav>` instead of `main`? It doesn't make sense, semantically.

Reply:
Good point.